src/index.js

```diff
@@ -0,0 +1,30 @@
+import _ from 'lodash'; //eslint-disable-line
+import './style.css';
+
+const todoList = document.getElementById('dynamic-list');
+const todos = [
+  {
+    description: 'Wash the dishes',
+    completed: false,
+    index: 0,
+  },
+  {
+    description: 'complete To Do list project',
+    completed: false,
+    index: 1,
+  },
+];
+
+function allTasks() {
+  todoList.innerHTML = '';
+  todos.forEach((todos) => {
+    todoList.innerHTML += `
+    <div class="one">
+     <input type="checkbox">
+     <p>${todos.description}</p>
+      <i class="fas fa-ellipsis-v"></i>
+    </div>
+    `;
+  });
+}
```

Comment:
- [ ] Please make sure the tasks appear in order of the `index` values for each task. 
For example, the apple appears first on the list because its index is `1`, and orange is second. It helps a lot when you connect it with localStorage
```
const todos = [
  {
    description: 'orange',
    completed: false,
    index: 2,
  },
  {
    description: 'apple',
    completed: true,
    index: 1,
  },
];
```

Reply:
Thank you for the review, i have fixed the issues